src/main/java/com/imaee/propinq/users/data/models/RealEstate.java

```diff
@@ -0,0 +1,29 @@
+package com.imaee.propinq.users.data.models;
+
+import com.imaee.propinq.users.data.enums.Role;
+import jakarta.persistence.*;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import org.hibernate.annotations.GenericGenerator;
+
+import java.util.UUID;
+
+@Entity(name = "real_states")
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+@Data
+public class RealEstate extends User{
+
```

Comment:
Faltaría un realEstateId
```suggestion
     @Id
     private final UUID realEstateId = UUID.randomUUID();
```